rllib/algorithms/algorithm_config.py

```diff
@@ -480,6 +480,12 @@ def __init__(self, algo_class=None):
         self.log_sys_usage = True
         self.fake_sampler = False
         self.seed = None
+        # TODO (sven): Remove this setting again in the future. We only need this
```

Comment:
Interesting. So there occur OOMs?

Reply:
It does seem to happen (nothing confirmed yet) when using the `evaluate_parallel_to_training` option, however, the leak does NOT scale with the number of eval workers. Which makes us believe it could be caused simply by the `training_step` being called in a thread. This setup will further strengthen this hypothesis down, if we don't see any change in the leaking using this option (and no eval workers at all).